src/amo/constants.js

```diff
@@ -358,3 +358,5 @@ export const WEBPACK_ENTRYPOINT = APP_NAME;
 export const ONE_YEAR_IN_SECONDS = 31536000;
 
 export const APPVERSION_FOR_ANDROID = '117.0';
+
+export const MZA_LAUNCH_DATETIME: Date = new Date(2023, 10, 1, 19, 0, 0, 0);
```

Comment:
I think the date constructor parses string values, e.g. `new Date("November 1, 2023 19:00:00")`, which has the benefit of being human readable.

Now with that said, this date is going to be evaluated against the local timezone I think, which means the switch to the new FxA name is going to happen for everyone at 7pm _their_ time. Is this intended? I would think that a UTC date would be better here.